MonkeyLoader.Resonite.Integration/UI/Inspectors/DefaultBuildMemberEditorHandlers.cs

```diff
@@ -0,0 +1,112 @@
+using FrooxEngine;
+using FrooxEngine.UIX;
+using HarmonyLib;
+using MonkeyLoader.Events;
+using MonkeyLoader.Patching;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MonkeyLoader.Resonite.UI.Inspectors
+{
+    [HarmonyPatch(typeof(SyncMemberEditorBuilder))]
+    [HarmonyPatchCategory(nameof(DefaultBuildMemberEditorHandlers))]
+    internal sealed class DefaultBuildMemberEditorHandlers : ResoniteMonkey<DefaultBuildMemberEditorHandlers>,
+        ICancelableEventHandler<BuildArrayEditorEvent>, ICancelableEventHandler<BuildBagEditorEvent>,
+        ICancelableEventHandler<BuildListEditorEvent>, ICancelableEventHandler<BuildPlaybackEditorEvent>,
+        ICancelableEventHandler<BuildFieldEditorEvent>, ICancelableEventHandler<BuildObjectEditorEvent>
+    {
+        public int Priority => HarmonyLib.Priority.Normal;
+
+        public bool SkipCanceled => true;
+
+        void ICancelableEventHandler<BuildArrayEditorEvent>.Handle(BuildArrayEditorEvent eventData) => throw new NotImplementedException();
```

Comment:
Why does this throw?

Reply:
Oh, woops, that should have been implemented like the others lol